Marlin/src/module/motion.cpp

```diff
@@ -1349,12 +1349,12 @@ void homeaxis(const AxisEnum axis) {
     // so here it re-homes each tower in turn.
     // Delta homing treats the axes as normal linear axes.
 
-    // retrace by the amount specified in delta_endstop_adj + additional 0.1mm in order to have minimum steps
+    // retrace by the amount specified in delta_endstop_adj + additional dist in order to have minimum steps
     if (delta_endstop_adj[axis] * Z_HOME_DIR <= 0) {
       #if ENABLED(DEBUG_LEVELING_FEATURE)
         if (DEBUGGING(LEVELING)) SERIAL_ECHOLNPGM("delta_endstop_adj:");
       #endif
-      do_homing_move(axis, delta_endstop_adj[axis] - 0.1 * Z_HOME_DIR);
+      do_homing_move(axis, delta_endstop_adj[axis] - MIN_STEPS_PER_SEGMENT / planner.axis_steps_per_mm[axis] * Z_HOME_DIR);
```

Comment:
Now I remember why this is here. Because if it's less than `MIN_STEPS_PER_SEGMENT` there would be no move. Hopefully this works. It's possible for float rounding to cause the mm result to be less than `MIN_STEPS_PER_SEGMENT` once it arrives in the planner.

Reply:
No complains so far by people who have been using this

Reply:
To see the rounding issue would be both random and rare.

Reply:
We could use (MIN_STEPS_PER_SEGEMENT + 1) to make sure. One stepper step loss in build height is not such a huge loss.